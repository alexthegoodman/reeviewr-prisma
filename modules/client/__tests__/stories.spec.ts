// const storyshots = require("@storybook/addon-storyshots");

// storyshots.default({
//   storyKindRegex: /^((?!.*?DontTest).)*$/,
//   storyNameRegex: /^((?!.*?DontTest).)*$/,
//   // test: storyshots.snapshotWithOptions({
//   //   snapshotSerializers: [(a: any) => sleep(0).then(() => a)],
//   // } as any),
//   configPath: `${__dirname}/../../.storybook`
// });
