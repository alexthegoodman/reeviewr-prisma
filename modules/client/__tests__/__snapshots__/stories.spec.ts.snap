// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Forms/LoginForm Title One 1`] = `
<section
  className="form"
>
  <h6>
    Login Forms 
    Title One
  </h6>
</section>
`;

exports[`Storyshots Forms/LoginForm Title Two 1`] = `
<section
  className="form"
>
  <h6>
    Login Forms 
    Title Two
  </h6>
</section>
`;
